addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -3,20 +3,56 @@ odoo.define('mail.NotificationAlert', function (require) {
 
 var Widget = require('web.Widget');
 var widgetRegistry = require('web.widget_registry');
+var core = require('web.core');
 
+var _t = core._t;
```

Comment:
`const { _t } = require('web.core');`

Also alphabetical order by module name, so this should be above `'web.Widget'`.